src/sentry/utils/prompts.py

```diff
@@ -19,6 +19,7 @@
     "data_consent_banner": {"required_fields": ["organization_id"]},
     "data_consent_priority": {"required_fields": ["organization_id"]},
     "issue_replay_inline_onboarding": {"required_fields": ["organization_id", "project_id"]},
+    "issue_feedback_hidden": {"required_fields": ["organization_id", "project_id"]},
```

Comment:
what if we renamed it to something like `issue_user_feedback`?  the trend seems to be that the names should be more related to the component rather than the action

Reply:
i think all `prompts` are used for hiding/dismissing something tho, it's implied but im not super adamant on it